### ISSUE TITLE
Редкое подвисание на 5 секунд при запуске после "Joystick: SDL GameController API"

### ISSUE THREAD

**Issue by Aynekko** [FWGS/xash3d-fwgs#1803]
Очень редко, но бывает зависание на 3-5 секунд при старте движка.
В консоли висит строчка Joystick: SDL GameController API.
Наверное, выгружается из оперативки или еще что-то.
Джойстика у меня нет. Если это нормальное поведение, просьба закрыть ишшуй.

**Comment by a1batross** [comment]
Будет ли подвисать если запускать с параметром `-noenginejoy`?

**Comment by Aynekko** [comment]
Потестирую в течение недели, отпишусь. На данный момент строчка "Joystick: SDL GameController API" пропала с этим параметром.

**Comment by a1batross** [comment]
Да, с ним встроенная поддержка геймпадов не активируется вообще.

**Comment by Aynekko** [comment]
Перечитал свое первое сообщение и наверное я не совсем правильно выразился.
Подвисает на 3-5 секунд и далее движок загружается в обычном режиме. Просто увеличивается время загрузки, а не виснет намертво.

**Comment by a1batross** [comment]
Я понял, да. Между строчкой про геймпад и Touch_Init как раз инициализация подсистемы геймпадов в SDL2.

**Comment by a1batross** [comment]
Поэтому чтобы быть уверенным что это где-то у нас или в SDL2 ошибка, я и попросил попробовать с -noenginejoy

**Comment by Vladislav4KZ** [comment]
У меня нет зависаний на строчке ``Joystick: SDL Game Controller API``.

Проверял в Half-Life 25th anniversary update с оригинальными client.dll и hl.dll

Проверял на:
Xash3D FWGS v49/0.21
Номер сборки: 3431
Тип сборки: win32-i386
Коммит: https://github.com/FWGS/xash3d-fwgs/commit/57774315774ef646951ec3e67d2171e36a3590a9
Ветка: master

ОС: Windows 10 Pro x64 22H2
Номер сборки ОС: 19045.4780

**Comment by Aynekko** [comment]
Вроде бы больше подвисаний не было.

**Comment by a1batross** [comment]
Я вполне могу предположить что SDL2 где-то подвисает при доступе к системным API и перечислении подключенных геймпадов, опроса драйверов и так далее.

Асинхронно бы это как-то делать.
